rspec-lambda/action.yml

```diff
@@ -20,6 +22,8 @@ runs:
         name: ${{ inputs.artifact-name }}
         path: public/js
     - name: RSpec tests
-      uses: docker://ghcr.io/university-of-york/faculty-dev-docker-images/ci/aws-lambda-ruby-postgres:2.7
```

Comment:
Is there any major negative side-effect to not using our lambda image for testing?

Reply:
It still runs in the dev lambda (example setup is at [faculty-dev-staff-manager/.github/workflows/test.yaml](https://github.com/university-of-york/faculty-dev-staff-manager/blob/4b3438b2bbce1ed53dd0ea46ffb03e0132269d05/.github/workflows/test.yaml#L25-L45)

This image doesn't have the exact setup of the lambda, it has a few more libraries installed, but should be very similar.